draft-ietf-quic-transport.md

```diff
@@ -2215,6 +2220,16 @@ Offset:
   on a stream - the sum of the re-constructed offset and data length - MUST be
   less than 2^64.
 
+Related Stream ID:
+
+: A 32-bit value containing the stream ID of a stream in the opposite direction.
+  This field is only present if the Offset field is zero length.  Using a
```

Comment:
Well, it's actually only present if the `R` bit is set.  But what I think you mean is that the `R` bit MUST NOT be set if the `OO` bits aren't zero, which doesn't belong in the definition of this field.

(Side-note:  This means these three bits can encode eight states, of which only five are defined valid.)